task_manager/task_manager.hh

```diff
@@ -0,0 +1,112 @@
+#pragma once
+
+#include "replica/database.hh"
+
+class task_manager {
+public:
+    static const size_t modules_count = 2;
+    enum class module {
```

Comment:
s/module/module_type/ so it won't be confused with potentially a `class task_manager::module`